test-built

```diff
@@ -1,11 +1,6 @@
 #!/bin/bash
 
-cd dist
-if [ ! -d bundle ]
```

Comment:
Ah, yea so the thought was to just build everything into meteor_core and then have de-meteorizer create a proper bundle. This is unpacked already.

Speaking of I don't think I properly explained the context when I submitted it :laughing: I was spitballing this with @tomitrescak in another issue.

If having creating a meteor CLI tarball for build is the desired output then i'll have to change other because my assumption was de-meteorizer can build it (using the deploy script) for those special cases where you SSH your own code up (most use a service deploy script).

I don't think you can safely deploy the tarball the meteor CLI gives you, at least it used to make you jump through hoops to make sure it was consistent on your machine vs the server.

Also for some reason I can't really see the code lines mentioned above so it's kind of hard to see which lines you're referring to.

 At any rate let me know what your thoughts are for the end goal, I just though i'd start with a generic version of what i'm using now.